libs/design-system/package.json

```diff
@@ -26,7 +27,7 @@
     "build:esm": "cross-env node_modules/.bin/tsc -p tsconfig.esm.json",
     "build:esm:watch": "cross-env node_modules/.bin/tsc -p tsconfig.esm.json -w --preserveWatchOutput",
     "build:types": "tsc --declaration --emitDeclarationOnly --declarationMap --declarationDir dist/types -p tsconfig.json",
```

Comment:
To do a "proper" [full monorepo](https://panda-css.com/docs/guides/component-library#use-panda-as-external-package) setup was going to take more time and was out of scope for this release, so that will have to be a future iteration. For now, we're simply exposing a Preset which will be the basis of the next phase no matter what direction we go.

Reply:
yeah, thanks I was just curious :)